### DISCUSSION TITLE
Publish a NextJS project as npm package to be consumed by another NextJS project

### DISCUSSION THREAD

**Discussion by andreas-groos** [vercel/next.js#31925] [Help]
I've been working on a project bootstrapped with create-next-app (with TS). I would like to publish it as an NPM package so that I can use the components as well as utility functions etc in another project.
I know how to publish to NPM but am unsure what modifications I have to make to webpack etc for it to succeed.

**Comment by lfvjimisola** [comment]
Have you solved this?
